src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -84,7 +86,7 @@ public SubmissionApiDelegateImpl(
 
         logger.info("Successfully got cso account information");
 
-        if (csoAccountDetails != null && !csoAccountDetails.getHasEfileRole()) {
+        if (accountDetails != null && !accountDetails.isEfileRolePresent()) {
             logger.warn("User does not have efiling role, therefore request is rejected.");
```

Comment:
out of scope: does this warn cover the logic of the if completely?

Reply:
yes because of the && right, the account details exists but the account does not have efile role

Reply:
barf. I suck today.